src/components/TransactionsList/TransactionsList-container.tsx

```diff
@@ -25,103 +20,58 @@ const headerCols: HeaderColsInterface[] = [
-    setSelectedRow(data);
-    setOpen(true);
-  };
-
-  const tablePaginationProps = {
-    rowsPerPageOptions: [15, 25, 50, 100, 250],
-    colSpan: 3,
-    count: props.blokchain.length,
-    rowsPerPage,
-    page,
-    SelectProps: {
-      inputProps: { 'aria-label': 'rows per page' },
-      native: true
-    },
-    onChangePage: handleChangePage,
-    onChangeRowsPerPage: handleChangeRowsPerPage,
-    ActionsComponent: TransactionListPagination
-  };
+  let [pageNumber, setPageNumber]: [number, Function] = React.useState(1);
 
   const renderTransactionListHeader = (headerCols: HeaderColsInterface[]) =>
     (headerCols.map((row: HeaderColsInterface) => (
       <TableCell
         key={row.id}
         align={row.numeric ? 'right' : 'left'}
         padding={row.disablePadding ? 'none' : 'default'}
-        sortDirection={orderBy === row.id ? order : false}
+        className={classes.td}
       >
-        <Tooltip
-          title="Sort"
-          placement={row.numeric ? 'bottom-end' : 'bottom-start'}
-          enterDelay={300}
-        >
-          <TableSortLabel
-            active={orderBy === row.id}
-            direction={order}
-            onClick={() => handleRequestSort(row.id)}
-          >
-            {row.label}
-          </TableSortLabel>
-        </Tooltip>
+        {row.label}
       </TableCell>
     )));
 
-  const renderTransactionListRows = (blokchain: Block[]) =>
-    (stableSort(
-      blokchain.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage),
-      getSorting(order, orderBy)
-    ).map((row: Block, index: number) => (
-      <TableRow onClick={() => handleClickOpen(row)} hover key={`Transaction${index}`}>
-        <TableCell>{row.amount}</TableCell>
-        <TableCell component="th" scope="row">{timestampToDate(row.timestamp)}</TableCell>
-        <TableCell>Exchange rate</TableCell>
-        <TableCell>Description</TableCell>
+  const renderTransactionListRows = (transactions: Transaction[]) => (
+    transactions.map((row: any, index: number) => (
+      <TableRow key={index + row.timestamp}>
+        <TableCell className={classes.td}>{row.value}</TableCell>
+        <TableCell className={classes.td} scope="row">{timestampToDate(row.timestamp)}</TableCell>
+        <TableCell className={classes.td}>{row.gasPrice || 'no info'}</TableCell>
+        <TableCell className={classes.td}>{row.description}</TableCell>
       </TableRow>
     )));
 
-  const modalDetailsProps: ModalDetailsProps = {
-    open,
-    handleClose: () => setOpen(false),
-    data: selectedRow
-  };
+  const handleScroll = (target: HTMLBodyElement) => {
+    if (target.scrollTop + target.clientHeight >= target.scrollHeight - 30) {
+      setPageNumber(pageNumber++);
+    }
+  }
+
+  const checkWalletHashAndFetchTransactions = (walletHash: any, pageNumber: number) => {
+    if (walletHash) {
+      actions.fetchEthereumTransactions({ walletHash: walletHash, page: pageNumber });
+    }
+  }
+
```

Comment:
I know it may seem obvious what is happening here for now, but it won't in a couple of weeks / months.
Maybe some comment?